pew/pew.py

```diff
@@ -188,7 +190,9 @@ def shell(env, cwd=None):
         if 'CMDER_ROOT' in os.environ:
             shell = 'Cmder'
         elif windows:
-            shell = 'powershell'
+            c = delegator.run('tasklist /fi "PID eq {0}" /fo csv /nh'.format(os.getppid()))
+            l = c.out
```

Comment:
I guess the `.out` can be moved to the line above, since we don't need `c` anywhere else. Renaming `l` to `lines` would also make it clearer, I think